src/guides/v2.3/config-guide/revert-split-database.md

```diff
@@ -0,0 +1,72 @@
+---
+group: configuration-guide
+title: Configure a single database from a split database
+ee_only: true
+functional_areas:
+  - Configuration
+  - System
+  - Setup
+---
+
+The [split database]({{ page.baseurl }}/config-guide/multi-master/multi-master.html) feature was deprecated in version 2.4.2 of {{ site.data.var.ee }} and {{ site.data.var.ce }}. Follow these instructions to revert from a split database to a single database implementation.
+
+## Revert a split database implementation
+
+Reverting from a split database to a single database implementation involves creating backups of the `magento_quote` and `magento_sales` databases before loading them into the single `magento_main` database.
+
+In this example, all three databases are installed on the same host named `magento2-mysql`.
+
+1. Create a backup of the `magento_quote` database:
+
+   ```bash
+   mysqldump -h "magento2-mysql" -u root -p magento_quote > ./quote.sql
+   ```
+
+1. Create a backup of the `magento_sales` database:
+
+   ```bash
+   mysqldump -h "magento2-mysql" -u root -p magento_sales > ./sales.sql
+   ```
+
+1. Load the `magento_quote` database into the `magento_main` database:
+
+   ```bash
+   mysql -h "magento2-mysql" -u root -p magento_main < ./quote.sql
+   ```
+
+1. Load the `magento_sales` database into the `magento_main` database:
+
+   ```bash
+   mysql -h "magento2-mysql" -u root -p magento_main < ./sales.sql
+   ```
+
+1. Drop the `magento_sales` database:
+
+   ```bash
+   mysql -h "magento2-mysql" -u root -p -e "DROP DATABASE magento_sales;"
+   ```
+
+1. Drop the `magento_quote` database:
+
+   ```bash
+   mysql -h "magento2-mysql" -u root -p -e "DROP DATABASE magento_quote;"
+   ```
+
+1. Remove the deployment configuration for `checkout` and `sales` in the `connections` section of the `env.php` file.
+1. Remove the deployment configuration for `checkout` and `sales` in the `resources` section of the `env.php` file.
+1. Restore foreign keys:
+
+   ```bash
+   bin/magento setup:upgrade
+   ```
+
+## Verify your work
+
+To verify that your single database implementation is working properly, perform the following tasks and verify that data is added to the `magento_main` database tables using a database tool like [phpMyAdmin]({{ page.baseurl }}/install-gde/prereq/optional.html#install-optional-phpmyadmin):
+
+1. Verify that foreign keys have been restored. For example, the `QUOTE_STORE_ID_STORE_STORE_ID` key in the `quote` database table.
+1. Verify that customers can place orders from the storefront.
+1. Verify that orders created before reverting the split database to a single database are available in the Admin.
+
+{:.bs-callout-tip}
+After configuration, the single `magento_main` database should contain 435 tables.
```

Comment:
This number (435) is not going to age well, and will likely vary, depending on the installed extensions. I recommend finding another way to visually confirm the procedure. Check that the migrated tables have content, maybe?

Reply:
Good point @keharper. If @paliarush doesn't have a better recommendation, I'll just remove this line.

Reply:
Oh. I see this is symlinked. The number won't be the same between releases anyway.